ompi/mca/osc/rdma/osc_rdma_component.c

```diff
@@ -999,11 +990,9 @@ static int ompi_osc_rdma_query_accelerated_btls (ompi_communicator_t *comm, ompi
     mca_bml_base_endpoint_t *base_endpoint;
     char **btls_to_use;
 
-    if (module) {
-        ompi_osc_rdma_selected_btl_insert(module, NULL, 0);
-        module->btls_in_use = 0;
-        module->use_memory_registration = false;
-    }
+    ompi_osc_rdma_selected_btl_insert(module, NULL, 0);
```

Comment:
In previous code, the function was sometime called with module being NULL (during initialization phase, when module was not created). So that usage is gone?